packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/servers/servers.constants.js

```diff
@@ -14,7 +14,7 @@ export const DC_2_ISO = {
   VIN: 'US',
   WAW: 'PL',
   YNM: 'IN',
-  YYZ: 'CA',
+  TOR: 'TOR',
```

Comment:
Should it not remain `CA` here ?

```suggestion
  TOR: 'CA',
```

Reply:
The macro region should be TOR.

Reply:
But this map gives the country, no ?

Reply:
@lionel95200x Could you check this please ? To me this should be CA here since it maps the DC to the country. Thanks :)